### PR TITLE
Branch docs for Foreman 2.5

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

**Comment by lzap** [comment]
Thank you, I suggest to merge this only after the official doc with headline features is merged: https://github.com/theforeman/theforeman.org/pull/1822

**Comment by upadhyeammit** [comment]
@lzap theforeman/theforeman.org#1822 has been merged, can we merge this one?
Thank you!

File: web/content/2.5.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+* Marek Hulan
+* Nagoor Shaik
+* Oleh Fedorenko
+* Ondřej Ezr
+* Partha Aji
+* Ranjan Kumar
+* Ron Lavi
+* Samir Jha
+* Shimon Shtein
+* Tomer Brisker
```
Comment (lzap) [inline web/content/2.5.md:99]: This is not the complete list.

**Comment by lzap** [comment]
Would you mind pulling the more updated document from https://github.com/theforeman/theforeman.org/pull/1825 ?

File: web/content/2.5.md (hunk 1 above)
Comment (upadhyeammit) [reply web/content/2.5.md:373]: Updated the list

**Comment by upadhyeammit** [comment]
> Would you mind pulling the more updated document from [theforeman/theforeman.org#1825](https://github.com/theforeman/theforeman.org/pull/1825) ?

Done!

**Comment by lzap** [comment]
Last thing and we are merging, can you also update the .github/PULL_REQUEST_TEMPLATE.md ?

**Review by lzap — APPROVED** [review]
Thank you!

**Pull request merged**